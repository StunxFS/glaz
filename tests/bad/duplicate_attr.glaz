@[copy; copy]
struct X {}

fn main() {
}
