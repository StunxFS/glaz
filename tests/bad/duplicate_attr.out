tests/bad/duplicate_attr.glaz:1:10: error: duplicate `copy` attribute
     | 
   1 | @[copy; copy]
     |         ^^^^
glaz error: could not compile `duplicate_attr`, aborting due to previous error.
